sdk/src/cameras/ad-96tof1-ebz/camera_96tof1.cpp

```diff
@@ -90,12 +95,30 @@ aditof::Status Camera96Tof1::initialize() {
     }
 
     // Initialize EEPROM
+    auto iter = std::find_if(m_devData.eeproms.begin(), m_devData.eeproms.end(),
+                             [](const EepromConstructionData &eData) {
+                                 return eData.driverName == "24c1024";
```

Comment:
`EEPROM_NAME` is already defined as `"24c1024"` in `target_definitions.h`. It would be more readable to use that define instead of the hard-coded value.

Reply:
target_definitions.h is part of the implementation of the low level API. The low level API doesn't need to expose the internals. The camera knows what EEPROM it is using. The best I can do is to declare a variable that holds the EEPROM name.